cp1404practicals/prac_09/silver_service_taxi_test.py

```diff
@@ -0,0 +1,6 @@
+from silver_service_taxi import SilverServiceTaxi
+
+taxi = SilverServiceTaxi("Aston Martin Taxi", 100, 2)
+taxi.drive(18)
+print(taxi)
+print(taxi.get_fare())
```

Comment:
You could use f string printing to incorporate a message when displaying the results for some clarity. Not an issue here I'd say but if you were testing a lot of different things having a message saying what everything is when it prints to the console would be beneficial